Add 11.20 release notes (#341)
mondoohq/docs: releases/2024-09-03-mondoo-11.20-is-out.md

---
slug: mondoo-11.20-is-out/
title: Mondoo 11.20 is out!
description: Announcing the 11.20 release of Mondoo with improved display of compliance evidence, expanded resources, and more!
authors: [tim]
image: /img/releases/2024-09-03-mondoo-11.20-is-out/query.png
tags: [release, mondoo]
---

## 🥳 Mondoo 11.20 is out! This release includes improved display of compliance evidence, expanded resources, and more!

Get this release: [Installation Docs](https://mondoo.com/docs/cnspec/) | [Package Downloads](https://releases.mondoo.com/cnspec/) | [Docker Container](https://hub.docker.com/r/mondoo/cnspec)

---

## 🧹 IMPROVEMENTS

### Quickly review compliance evidence

> View compliance evidence gathered by Mondoo, all without having to leave Compliance Hub. Compliance Hub query pages now show evidence for each scanned asset directly on the page, so you can quickly review evidence without needed to view each asset invididually.

Reviewer: ```suggestion
View compliance evidence gathered by Mondoo leaving Compliance Hub. Now Compliance Hub query pages show evidence for each scanned asset right on the page, so you can quickly review evidence without individually opening each asset.
```

---
slug: mondoo-11.20-is-out/
title: Mondoo 11.20 is out!
description: Announcing the 11.20 release of Mondoo with improved display of compliance evidence, expanded resources, and more!
authors: [tim]
image: /img/releases/2024-09-03-mondoo-11.20-is-out/query.png
tags: [release, mondoo]
---

## 🥳 Mondoo 11.20 is out! This release includes improved display of compliance evidence, expanded resources, and more!

Get this release: [Installation Docs](https://mondoo.com/docs/cnspec/) | [Package Downloads](https://releases.mondoo.com/cnspec/) | [Docker Container](https://hub.docker.com/r/mondoo/cnspec)

---

> 
> ## 🎉 NEW FEATURES

Reviewer: ```suggestion
```
